### PR TITLE
examples: update usage() with DTLSv1.3 version

### PR DESCRIPTION
**Pull request by rizlik**
# Description

Update help message of example/client and example/server to include DTLSv1.3

### CODE REVIEW

**Comment by rizlik** [comment]
@miyazakh can you please add to the Japanese part as well?

**Comment by rizlik** [comment]
> @miyazakh can you please add to the Japanese part as well?

@miyazakh will open a separate PR for the Japanese line

File: examples/client/client.c [hunk 1]
```diff
@@ -1166,7 +1166,12 @@ static const char* client_usage_msg[][70] = {
         "-e          List Every cipher suite available, \n",            /* 19 */
         "-g          Send server HTTP GET\n",                           /* 20 */
         "-u          Use UDP DTLS,"
+#ifndef WOLFSSL_DTLS13
```
Comment (dgarske) [inline examples/client/client.c:1169]: Can you fix the idents so they match? Does the SCTP DTLS section below need updated too (guessing no)?

**Review by dgarske — CHANGES_REQUESTED** [review]

File: examples/server/server.c [hunk 2]
```diff
@@ -829,7 +829,12 @@ static const char* server_usage_msg[][64] = {
         "-b          Bind to any interface instead of localhost only\n",/* 13 */
         "-s          Use pre Shared keys\n",                            /* 14 */
         "-u          Use UDP DTLS,"
-           " add -v 2 for DTLSv1, -v 3 for DTLSv1.2 (default)\n",       /* 15 */
+#ifndef WOLFSSL_DTLS13
```
[the remark this answers is not in the document]
Comment (rizlik) [reply examples/server/server.c:832]: indent fixed. 
No, we do not support SCTP yet.

**Pull request merged**